documentation/HandsOnPropertyBased.md

```diff
@@ -109,13 +109,12 @@ import * as fc from 'fast-check';
 import { sort } from '../src/sort';
 
 test('should contain the same items', () => {
-  const count = (tab, element) => tab.filter(v => v === element).length;
   fc.assert(
     fc.property(fc.array(fc.integer()), data => {
       const sorted = sort(data);
-      expect(sorted.length).toEqual(data.length);
+      assert(sorted.length === data.length);
       for (const item of data) {
-        expect(count(sorted, item)).toEqual(count(data, item));
+        assert(sorted.includes(item));

```

Comment:
The check is not totally equivalent. Before your change having `data = [1,2,2]` and `sorted = [1,1,2]` would have been wrong. Following your change it passes.

Reply:
Shucks! You've clearly thought longer about this than I have. Thanks.